### ISSUE TITLE
Update to 1.8: postinst: cannot create /root/.ssh/id_rsa: Permission denied

### ISSUE THREAD

**Issue by silopolis** [OpenMediaVault-Plugin-Developers/openmediavault-omvextrasorg#5]
Hi,

This error appeared on last plugin update:
```
Setting up openmediavault-omvextrasorg (1.8) ...
OK
[...]
OK
/var/lib/dpkg/info/openmediavault-omvextrasorg.postinst: 160: /var/lib/dpkg/info/openmediavault-omvextrasorg.postinst: cannot create /root/.ssh/id_rsa: Permission denied
dpkg: error processing openmediavault-omvextrasorg (--configure):
 subprocess installed post-installation script returned error exit status 2
Processing triggers for openmediavault ...
[...]
Errors were encountered while processing:
 openmediavault-omvextrasorg
```

Looking at the postinst script it seems it wants to set the same root ssh keys on all installs ?!!
Shouldn't it create special keys like "omvextras_id_rsa" and shouldn't these be stored elsewhere than in "/root/.ssh/", like "/etc/openmediavault" or "/home/openmediavault/.ssh/" or may be even better "/etc/omvextras/" ?

Supportinfo log upload should be able to use a specified keys and not the root's ones.

Bests

**Comment by ryecoaaron** [comment]
I find it strange that the script couldn't write the files in /root/.ssh/.  Do you have them locked?

All problems should be fixed in 1.9

**Comment by silopolis** [comment]
OK, 1.9 installed without a problem and created the keys in /etc/omvestras... nice, thank you for the fast fix !

About my root keys, no, I don't seem to have them locked, or they are in a way I'm not aware of !

```
# ls -la /root/.ssh/
...
-rw------- 1 root root  672 Dec 20 10:04 id_dsa
-rw------- 1 root root 1679 Dec 20 10:04 id_rsa
...
```
